fe/fe-core/src/main/java/org/apache/doris/analysis/BinaryPredicate.java

```diff
@@ -483,11 +483,13 @@ public Pair<SlotRef, Expr> extract() {
     public void analyzeImpl(Analyzer analyzer) throws AnalysisException {
         super.analyzeImpl(analyzer);
         this.checkIncludeBitmap();
-        // Ignore placeholder
-        if (getChild(0) instanceof PlaceHolderExpr || getChild(1) instanceof PlaceHolderExpr) {
+        // Ignore placeholder, when it type is invalid.
+        // Invalid type could happen when analyze prepared point query select statement,
+        // since the value is occupied but not assigned
+        if ((getChild(0) instanceof PlaceHolderExpr && getChild(0).type == Type.UNSUPPORTED)
```

Comment:
Do you mean add setting type of placeholder to UNSUPPORTED to represent normal prepared statement and INVALID for point query?
If yes, it's somewhat tricy.

Reply:
not just point query for FULL_PREPARED type of prepared statement